AppCreationScripts/AppCreationScripts.md

```diff
@@ -0,0 +1,147 @@
+# Registering the sample apps with Microsoft identity platform and updating the configuration files using PowerShell scripts
+
+## Overview
+
+### Quick summary
+
+1. On Windows run PowerShell and navigate to the root of the cloned directory
+1. In PowerShell run:
+   ```PowerShell
+   Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope Process -Force
+   ```
+1. Run the script to create your Azure AD application and configure the code of the sample application accordingly. (Other ways of running the scripts are described below)
+   ```PowerShell
+   cd .\AppCreationScripts\
+   .\Configure.ps1
+   ```
+1. Open the Visual Studio solution and click start
```

Comment:
General comment: Add relevant punctuation at the end of each bullet/item.

e.g.

1. Open the Visul Studio solution and click start:
1. Run the script to create your Azure AD application and configure the code of the sample application accordingly (Other ways of running the scripts are described below):

Reply:
My apologies, I haven't read this through at all -will do.